documentation/src/docs/asciidoc/docinfos/index-docinfo.html

```diff
@@ -0,0 +1 @@
+<link rel="icon" type="image/png" href="{{ site.baseurl }}/img/junit5-logo.png">
```

Comment:
For the user-guide, I changed the docinfo
![image](https://cloud.githubusercontent.com/assets/8221614/26374737/0c8409ba-400f-11e7-8c1a-4c3b6828c399.png)

Regarding the javadocs, how should the logo be referenced since it is in the `asciidoc/doc`? The `prepareDocsForUploadToGhPages` generates both the userguide and the javadocs and here I have access to the logo. On the other hand, `aggregateJavaDocs` generates only the javadocs without providing access to the image. Should the image be copied to the output folder in `aggregateJavaDocs` as well?